Core.Collectors/Web/HttpClientWrapper.cs

```diff
@@ -48,6 +50,11 @@ public Task<HttpResponseMessage> GetAsync(string requestUrl, IAuthentication aut
 
         private async Task<HttpResponseMessage> MakeRequestAsync(string requestUrl, HttpMethod method, string requestBody, IAuthentication authentication, ProductInfoHeaderValue productInfoHeaderValue, string eTag)
         {
+            using Activity requestTrace = OpenTelemetryTracer.GetActivity(OpenTelemetryTrace.Request).Start();
+            requestTrace.AddTag("Url", requestUrl);
```

Comment:
Interesting, so the activity gets "committed" once it is out of scope (no longer accessible)?

Reply:
Yes, once an Activity is disposed it is emitted.